src/components/SideBar/index.tsx

```diff
@@ -159,6 +196,14 @@ const SideBar = () => {
             <div id="sidebar-tab" onClick={handleOnClick}>
                 <span id="sidebar-tab-icon">{openSidebar ? '◀' : '▶'}</span>
             </div>
+            <GhListButtonBar
+                id="ghlist"
+                as="a"
+                href="https://data.covid-19.global.health/"
```

Comment:
This url should be replaced with `.env` variable in order to use appropriate links in Dev and Prod environments